common-util/AddressLink/index.jsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { Tooltip } from 'antd/lib';
+import { GATEWAY_URL } from '@autonolas/frontend-library';
+import { getChainId } from 'common-util/functions';
+
+// *******************************************************
+// ***** MOVE EVERYTHING BELOW TO AUTONOLAS LIBRARY ******
+// *******************************************************
+
+export const getExplorerURL = (chainId = 1) => {
+  switch (chainId) {
+    case 5:
+      return 'https://goerli.etherscan.io';
+    case 137:
+      return 'https://polygonscan.com';
+    case 80001:
+      return 'https://mumbai.polygonscan.com';
+    case 100:
+      return 'https://gnosisscan.io';
+    case 10200:
+      return 'https://gnosis.blockscout.com';
```

Comment:
@DavidMinarsch these links are used for the UI output only, not for API RPC calls